runtime/doc/treesitter.txt

```diff
@@ -1119,9 +1121,11 @@ Query:iter_matches({node}, {source}, {start}, {stop}, {opts})
     Parameters: ~
       • {node}    (`TSNode`) under which the search will occur
       • {source}  (`integer|string`) Source buffer or string to search
-      • {start}   (`integer`) Starting line for the search
-      • {stop}    (`integer`) Stopping line for the search (end-exclusive)
-      • {opts}    (`table?`) Options:
+      • {start}   (`integer?`) Starting line for the search. Defaults to
+                  `node:start()`.
+      • {stop}    (`integer?`) Stopping line for the search (end-exclusive).
+                  Defaults to `node:end_()`.
+      • {opts}    (`table?`) Optional keyword arguments:
                   • max_start_depth (integer) if non-zero, sets the maximum
                     start depth for each match. This is used to prevent
                     traversing too deep into a tree. Requires treesitter >=
```

Comment:
We also don't need this line anymore, since treesitter 0.20.9 is released

Reply:
Right. #27198 was merged. Thanks!